Data_Engineer_KoNoHa.ipynb

```diff
@@ -0,0 +1,2775 @@
+            }
+          },
+          "metadata": {},
+          "execution_count": 129
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Display summary statistics\n",
+        "df.describe()"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 300
+        },
+        "id": "AftiAVEuUN6I",
+        "outputId": "80ff9ab0-9e37-41a9-ec3c-f111c19a8d64"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "          Bathrooms     Car Parks\n",
+              "count  51870.000000  36316.000000\n",
+              "mean       3.073434      2.007848\n",
+              "std        1.631079      1.310427\n",
+              "min        1.000000      1.000000\n",
+              "25%        2.000000      1.000000\n",
+              "50%        2.000000      2.000000\n",
+              "75%        4.000000      2.000000\n",
+              "max       20.000000     30.000000"
```

Comment:
If this statistical value result can be round off into two decimal values, it will be easier to read!
My eyes hurtin' jkjk

Reply:
ok ok ok